tasks/storage_stack.yml

```diff
@@ -0,0 +1,35 @@
+- name: Get stack outputs
```

Comment:
This makes it look like new hosts in the inventory but would we ever be able to connect to them, given the storage stack will have its own private key?

Reply:
That's right, we cannot connect to them. We have to make sure we don't run tasks on them or gather facts, but we still use them for hostvars.